hpx/runtime/actions/continuation.hpp

```diff
@@ -153,10 +155,10 @@ namespace hpx { namespace actions
         virtual ~continuation() {}
 
         //
-        virtual void trigger() const;
+        virtual void trigger();
 
         template <typename Arg0>
-        inline void trigger(Arg0 && arg0) const;
+        inline void trigger(Arg0 && arg0);
 
         //
         virtual void trigger_error(boost::exception_ptr const& e) const;
```

Comment:
If the `trigger()` member functions are not const anymore, then `trigger_error()` shouldn't be const either.